mw-webapp/src/component/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,38 @@
+import {ReactNode} from "react";
+import clsx from "clsx";
+import {PositionTooltip} from "src/component/tooltip/positionTooltip/PositionTooltip";
+import styles from "src/component/tooltip/Tooltip.module.scss";
+
+interface TooltipProps {
+  /**
+  * Tooltip's children (ReactNode)
+  */
+  children: ReactNode;
+  /**
+  * Tooltip's content (text)
+  */
+  content: string;
+  /**
+  * Tooltip's class-style (selector css)
+  */
+  styleComponent: string;
```

Comment:
let's use name className and appropriate comment

Reply:
fixed